recipes/opene57/all/conanfile.py

```diff
@@ -39,10 +41,9 @@ def _min_cppstd(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
```

Comment:
Please, keep the previous configuration. The `Visual Studio` is still used in Conan 1.x. 

I don't get about gcc 7 to 8. Does 1.6.5 require GCC 8?

Reply:
It's not an hard requirement, but using gcc 8 as minimum version is suggested. It can be still kept as gcc-7.

I will submit the change with Visual Studio again. I thought that in conan2 it is now "msvc", that's why I removed it